go-controller/hybrid-overlay/pkg/controller/node.go

```diff
@@ -128,32 +129,44 @@ func NewNode(
 	return n, nil
 }
 
-// Run starts the controller
-func (n *Node) Run(stopCh <-chan struct{}) error {
+// Run starts the controller and does not return until all operations have
+// terminated after closing the stop channel
```

Comment:
"after the stop channel is closed"
(ie, it does not close the stop channel, it waits for someone else to do it)

Reply:
fixed